apps/dialogue/src/tasks/dialogue.tasks.handler.fields.prompt.ts

```diff
@@ -0,0 +1,74 @@
+import { PromptTemplate } from 'libs/llm/prompt/prompt.template';
+import { TaskFieldDto } from './store/dialogue.tasks.store.dto';
+
+export const taskFieldConditionPrompt = PromptTemplate.create<{
+  condition: string;
+  values: string;
+}>(
+  'task-field-condition',
+  `
+Evaluate CONDITION based on VALUES
+Return a parsable JSON object with format { "result": boolean }
+Never add explanation or comments.
+
+VALUES:
+<%= data.values %>
+
+CONDITION:
+<%= data.condition %>`,
+);
+
+export const taskFieldExpressionPrompt = PromptTemplate.create<{
+  values: string;
+  fieldPrompt: string;
+}>(
+  'task-field-expression',
+  `
+Given this JSON object:
+<%= data.values %>
+
+<%= data.fieldPrompt %>
+Return a parsable JSON object with format { "result": value }.
+Do not add notes or explanations.`,
+);
+
+export const taskFieldValidationPrompt = PromptTemplate.create<{
+  values: string;
```

Comment:
I am not sure I can read this call properly, but two things are bothering me:
- The presence of a "values" parameter (plural) which is not a list or collection
- The presence of both a "value" and a "values" parameters.
- "values" is not used here (but I guess it is used by PromptTemplate.create?)

Reply:
you are right, values is actually not used in the template and can be removed. thx!